components/resources/brave_shields_strings.grdp

```diff
@@ -13,24 +13,24 @@
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_UP" desc="Status label for when Brave Shields is enabled.">
-    Shields are UP <ph name="LINK_BEFORE">$1</ph>for<ph name="LINK_AFTER">$2</ph>
+    <ph name="STATUS_BEFORE">$1</ph>Shields are UP<ph name="STATUS_AFTER">$1</ph> for <ph name="HOST">$2</ph>
```

Comment:
I am a little concerned that we are reusing `$1` for both `BEFORE` and `AFTER` - typically you'd have `$2` in the second placeholder. It might be confusing to do this differently from all other GRD(P)s.

Reply:
Yup, I was a bit concerned about this too - would it help if I gave them both the same name?

The thing is, it makes life a lot easier on the parsing side because we can tell whether a placeholder is meant to wrap some text or just be inserted. (i.e. one or two occurrences of the placeholder).

For Brave stuff I'm planning on converting all our usages over to this pattern, if it lands.

Reply:
Easy as, lets see how it goes